ledger/apptxn_test.go

```diff
@@ -3629,8 +3629,8 @@ func TestUnfundedSenders(t *testing.T) {
 			dl.fullBlock()
 		}
 
-		fmt.Printf("addrs[0] = %+v\n", addrs[0])
-		fmt.Printf("addrs[1] = %+v\n", addrs[1])
+		t.Logf("addrs[0] = %+v\n", addrs[0])
+		t.Logf("addrs[1] = %+v\n", addrs[1])
```

Comment:
Not sure if these print statements are meant to be here or not, maybe @jannotti knows?

Reply:
I suspect they were debugging, and can be removed entirely.